EnglishFilter/sections/general_extensions.txt

```diff
@@ -3107,7 +3107,7 @@ saavn.com#%#!function(a,b){b=new MutationObserver(function(){a.classList.contain
 ! https://github.com/AdguardTeam/AdguardFilters/issues/5388
 csgoconfigs.com#%#document.cookie = "AD_SHOWN=true";
 ! https://forum.adguard.com/index.php?threads/littlebyte-net.22689/
-littlebyte.net#%#AG_onLoad(function(){AG_each('a[onclick^="getActionContent"]',function(a){a.setAttribute('href','#');})})
+littlebyte.net#%#//scriptlet('remove-attr', 'href', 'a[onclick^="getActionContent"]')
 littlebyte.net#%#//scriptlet('remove-attr', 'target', 'a[onclick^="getActionContent"]')
```

Comment:
What about this `littlebyte.net#%#//scriptlet('remove-attr', 'href|target', 'a[onclick^="getActionContent"]')`?

Reply:
Okay, I have changed it.